src/theory/arith/nl/cad/constraints.h

```diff
@@ -0,0 +1,97 @@
+/*********************                                                        */
+/*! \file constraints.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Gereon Kremer
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Implements a container for CAD constraints.
+ **
+ ** Implements a container for CAD constraints.
+ **/
+
+#ifndef CVC4__THEORY__NLARITH__CAD__CONSTRAINTS_H
```

Comment:
This guard should be `CVC4__THEORY__ARITH__NL__CAD__CONSTRAINTS_H` (namespaces caps with two underscores separating), similarly format the others.

Reply:
I've renamed the guards.